go/pkg/sysdb/coordinator/table_catalog.go

```diff
@@ -2000,6 +2000,20 @@ func (tc *Catalog) DeleteCollectionVersion(ctx context.Context, req *coordinator
 	return &result, nil
 }
 
+func (tc *Catalog) BatchGetCollectionVersionFilePaths(ctx context.Context, req *coordinatorpb.BatchGetCollectionVersionFilePathsRequest) (*coordinatorpb.BatchGetCollectionVersionFilePathsResponse, error) {
```

Comment:
this is not the pattern that we have rn. Ideally we convert these to internal types in collection_service.go and not expose proto types here

Reply:
there's a few methods in this file that accept the proto type directly, so I figured it was ok to avoid a layer of type conversions
I can add a type wrapper if you prefer

I would like to note that in general these sorts of type wrappers have bitten us several times in Go since if fields are omitted during struct init they're set to default values

Reply:
I mean you could just have a list of collection ids as param here? Don't need a new type

Reply:
yeah that works, will update